helm/reana/templates/reana-message-broker.yaml

```diff
@@ -5,13 +5,15 @@ metadata:
   name: {{ include "reana.prefix" . }}-message-broker
   namespace: {{ .Release.Namespace }}
 spec:
+  type: "NodePort"
   ports:
    - port: 5672
      targetPort: 5672
      name: "tcp"
      protocol: TCP
-   - port: 15672
+   - port: 32585
```

Comment:
Shouldn't this be done conditionally only, depending on `{{ .Values.debug.enabled }}`?  

Also, changing target port from 15672 to 32585 would require to amend also `r-m-broker`'s Dockerfile.  It is best to keep internallly 15672 and amend the exposing port only (and only when debug is set).  

Option 1: let's merge the PR without port and exposure changes and you'd build on top.  Option 2: please keep it but then amend the port scenario back to 15672 and add the conditional?

Reply:
I would go for option 2, testing it rn 👍